tests/test_flux_analysis/test_deletion.py

```diff
@@ -25,7 +25,7 @@ def test_single_gene_deletion_fba_benchmark(
 ) -> None:
     """Benchmark single gene deletion using FBA."""
     model.solver = all_solvers
-    benchmark(single_gene_deletion, model)
+    benchmark(single_gene_deletion, model, model.genes[1::10])
```

Comment:
Is there a special reason to start with the second gene?

Reply:
Nope, can start with the first.

Reply:
I later saw that it's done so in quite a few places. Let's just keep it.